flashcards/index.html

```diff
@@ -13,6 +13,15 @@
         margin: 0;
         padding: 0;
         font-family: sans-serif;
+
+        --color-button-primary-fg: white;
```

Comment:
Tiny suggestion: I'd go with `:root` selector, now those variables are defined "twice" in the dom.